packages/suite/src/views/wallet/send/components/Outputs/index.tsx

```diff
@@ -24,12 +23,31 @@ const Container = styled.div<{ $height: number }>`
     }
 `;
 
-const StyledCard = styled(Card)`
+const StyledCard = styled(Card)<{ isTopRadiusZero: boolean }>`
     gap: ${spacingsPx.xs};
+    ${props =>
+        props.isTopRadiusZero &&
+        css`
+            border-top-left-radius: 0;
+            border-top-right-radius: 0;
+        `}
 `;
 
-const StyledEvmExplanationBox = styled(EvmExplanationBox)`
-    margin-bottom: ${spacingsPx.md};
+const StyledEvmExplanationCard = styled(Card)`
+    background-color: #d5e9f1;
+    color: #00597f;
+    font-size: 14px;
+    flex-direction: row;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+    padding-top: 8px;
+    padding-bottom: 8px;
+    line-height: 16px;
+    align-items: center;
```

Comment:
```
    background-color:${({ theme }) => theme.lightAccentBlue300};
    color: ${({ theme }) => theme.lightAccentBlue700};
    ${typography.hint}
    flex-direction: row;
    border-bottom-left-radius: 0;
    border-bottom-right-radius: 0;
    padding-top: ${spacingsPx.xs};
    padding-bottom: ${spacingsPx.xs};
    align-items: center;
```

Reply:
Fixed in 55300502b35233c7764900e95425bc816ebff818
Used:
```
background-color: ${({ theme }) => theme.backgroundAlertBlueSubtleOnElevation0};
color: ${({ theme }) => theme.textAlertBlue};
```
(because those colors are associated to the keys `backgroundAlertBlueSubtleOnElevation0` and `textAlertBlue`